src/plugins/intel_cpu/src/nodes/transpose.cpp

```diff
@@ -174,6 +206,55 @@ void MKLDNNTransposeNode::createPrimitive() {
     }
 }
 
+void MKLDNNTransposeNode::createReorderPrimitive(const mkldnn::memory::desc& desc, void* srcPtr, void* dstPtr, bool isInput) {
+    mkldnn::primitive_attr attr;
+    const auto engine = getEngine();
+    MKLDNNMemoryPtr src_blocked = std::make_shared<MKLDNNMemory>(engine);
+    MKLDNNMemoryPtr dst_blocked = std::make_shared<MKLDNNMemory>(engine);
+
+    if (isInput) {
+        dst_blocked->Create(MKLDNNExtensionUtils::makeDescriptor(desc), dstPtr, false);
+
+        const auto newDims = dst_blocked->getStaticDims();
+        auto newDesc = mkldnn::memory::desc(MKLDNNExtensionUtils::convertToDnnlDims(newDims),
+                                               dst_blocked->GetDataType(), memory::format_tag::acdb);
+        src_blocked->Create(MKLDNNExtensionUtils::makeDescriptor(newDesc), srcPtr, false);
+    } else {
```

Comment:
I guess "else" branch is not used here so can be deleted

Reply:
Done